### DISCUSSION TITLE
Is there a way to use the store content-items outside of a YesSQL?

### DISCUSSION THREAD

**Discussion by MikeAlhayek** [OrchardCMS/OrchardCore#7845] [Q&A]
I have an existing databases what are very busy and used by many micro services. I would like to connect to us using OrchardCore and be able to manage the records. I don't want to eliminate YesSQL as it'll allow flexibility of new stuff. 

My thought are to create a module where I can per-define my content-type, content-field, and possible content-parts. I want to be able to create the CRUD view dynamically like Orchard already does. However, but may be store the content-Items outside YesSql. I already have my own stores and I am able to connect to them. I don't mind if the definitions of the content-types is stored in YesSQL for the display managers and the UI to work. But I would like to store the content-items/records using my stores. I don't also care about versioning but need the data stored using standard sql tables.

Is what I am trying to achieve possible? How can I accomplish such a thing?

I know I can create standard CRUD function using standard MVC. But, I want to be able to utilize the built in Display/Edit UI so I don't have to create custom views for everything I need. This way I can change/implement new themes easier and quicker.

**Comment by Piedone** [comment]
Check out [this issue](https://github.com/OrchardCMS/OrchardCore/issues/7455). While it's about sharing content items between tenants I think very similar questions arise here too: You could have placeholder (or "link") content items loading data from the external source.
